media-sound/jack-smf-utils/files/jack-smf-utils-1.0-unbundle-libsmf-and-smfsh.patch

```diff
@@ -0,0 +1,107 @@
+===============================================================================
```

Comment:
This is actually really nice in terms of information (I usually recommend people follow [this](https://dev.gentoo.org/~vapier/clean-patches)).

The format looks CVS-ish, we can keep the ===s if you like at the bottom, but we should cut out some lines (more to follow...)

Using git format-patch to generate them may be useful but unfortunately you may still have to cut out cruft. This isn't strictly necessary, it's just nice. The patch itself looks good by the way.

Reply:
Good read. You're right, the diff lines waste precious space I have removed them and will refer to the link.

I've done it myself before, trying to dig through a pile of patches when the file name or diff doesn't contain any context, it's painful. I can also understand the reason for trimming out pointless bytes - my bad.

Thanks for the kind words.